### PR TITLE
Hide the start page to improve test performance

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/VisualStudio/IntegrationTest/TestUtilities/InProcess/StartPage_InProc.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                return (int)property.Value == 4;
+            });
+        }
+
+        public void SetEnabled(bool enabled)
+        {
+            InvokeOnUIThread(() =>
+            {
+                var property = GetProperty();
+                property.Value = enabled ? 5 : 4;
```
Comment (jaredpar) [inline src/VisualStudio/IntegrationTest/TestUtilities/InProcess/StartPage_InProc.cs]: Where do these numbers come from?

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/TestUtilities/InProcess/StartPage_InProc.cs]: image

Comment (mavasani) [reply src/VisualStudio/IntegrationTest/TestUtilities/InProcess/StartPage_InProc.cs]: @sharwell Can we please add a comment?

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/TestUtilities/InProcess/StartPage_InProc.cs]: ➡️ Now updated

**Comment by sharwell** [comment]
@jaredpar for approval

**Pull request merged**